### ISSUE TITLE
Login stopped working

### ISSUE THREAD

**Issue by sprjr** [icbestCA/giftmanager#19]
During regular use (with about 10 or so users), logins stopped working. When entering correct credentials, the login page just reloads with no indication of any success, failure, or other change. Standard log output also provides no information about the login attempt other than something to the effect of the snippet below. No changes were made when it stopped working as it was in an "appliance" state where I didn't want to mess with it due to how regularly it's being used. A user notified me of the issue.

```
[07/Oct/2024 14:48:31] "GET /favicon.ico HTTP/1.1" 404 -
[07/Oct/2024 14:48:41] "POST /login HTTP/1.1" 200 -
```

I've also attempting to remove all backup files except for a single user.json and ideas.json file, backing up the rest of the data into another directory. Unfortunately this has had no observable impact on the behavior of the software.

**Comment by icbestCA** [comment]
Hello, sorry for the inconvenience, I've tested some things. When you get the code 200 for login it's that it wasn't successful (will be 302). That said, can I get more details on your current setup? Docker or flask? Are you using the argon2 or sha-1 hashing function? You can see this when opening the users.json and checking the hashes.
Thanks.

**Comment by sprjr** [comment]
It's docker, and I'm more than happy to provide my (admittedly basic) compose file here if you'd like. Looking at my users backup file it appears to be argon2.

**Comment by sprjr** [comment]
Also, is there any way to increase log verbosity? At least then I could continue to work on it.

**Comment by icbestCA** [comment]
Not really, it all comes from the flash messages. You could update with docker compose and get the latest version, I updated the login page to show the errors messages (user not existing or wrong password). This is what I suspect since you only get the 200 code.

**Comment by sprjr** [comment]
I've gone ahead and pulled and run the newest image; however, it now informs me that my "user doesn't exist" on the login page. I continue to get a http 200 code. I've checked and the `./backup/` directory and the relevant contents are all owned by my user and have 755 permissions.

**Comment by icbestCA** [comment]
Could you try to the default logins demo/password or user2/password2 I think the directory might not be mounted correctly and there is another users.json elsewhere with the default logins. 
I don't see other options for the issue.
Thanks.

**Comment by sprjr** [comment]
That worked. On login, I can see that only the two default users are listed. I'll exec into the container to see if there's anything unusual in that directory.

**Comment by sprjr** [comment]
Okay, I have figured it out. As with most issues, this was on me. I had the container's directory specified incorrectly. I'm not sure how this could have happened, as I had it working for weeks previous to this, but once I corrected it to `/app/data` (or whatever the default container volume is), copied the correct data into the file it was referencing, and restarted the container, it came back up.

Thank you for working with me, and I'm glad it was a "me" issue rather than anything else. Thank you for developing this.
